src/test/java/core/basesyntax/service/ReportMakerServiceTest.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.impl.ReportMakerServiceImpl;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ReportMakerServiceTest {
+    @Test
+    public void reporterMaker_report_ok() {
+        Map<String, Integer> base = new HashMap<>();
+        base.put("banana", 5);
+        base.put("apple", 25);
+        String expected = "fruit, quantity" + System.lineSeparator()
+                + "banana, 5" + System.lineSeparator()
+                + "apple, 25";
+        String actual = new ReportMakerServiceImpl().report(base);
```

Comment:
introduce `ReportMakerServiceImpl` as a field variable. Make this for all test cases.